xml/hardening.xml

```diff
@@ -4024,17 +4043,21 @@ RhostsRSAAuthentication no</screen>
         <link xlink:href="https://en.wikipedia.org/wiki/Address_space_layout_randomization"/>
        </para>
       </listitem>
+      <listitem>
+       <para>
+        <link xlink:href="https://en.wikipedia.org/wiki/NX_bit"/>
+       </para>
+      </listitem>
      </itemizedlist>
      <para>
       Since version 12, &productname; already comes with some buffer
-      overflow mitigation techniques being enabled by default. One of them
-      is Address Space Layout Randomization (ASLR).
+      overflow attack mitigation techniques being enabled by default.
      </para>
      <para>
-      ASLR should be enabled by default already. You can check the output of
-      the following command (should be 2):
+      ASLR is enabled by default. This can be verified with the output
+      of the following command. The expected result is 2:
      </para>
-<screen># cat /proc/sys/kernel/randomize_va_space
+<screen>&prompt.user; cat /proc/sys/kernel/randomize_va_space
```

Comment:
Two issues:

* Remove the space between `&prompt.user;` and `cat`.
* Wrap `cat` with `<command>`.